core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/PersonAccountEditViewModel.kt

```diff
@@ -0,0 +1,19 @@
+package com.ustadmobile.core.viewmodel
+
+data class PersonUsernameAndPasswordModel(
+
+    val username: String = "",
+
+    val currentPassword: String = "",
+
+    val newPassword: String = "",
+
+    val passwordConfirmed: String = "",
+)
+
+data class PersonAccountEditUiState(
+
+    val personUsernameAndPassword: PersonUsernameAndPasswordModel = PersonUsernameAndPasswordModel(),
+
+    val currentPasswordError: String = "",
```

Comment:
All error strings must be nullable. Any non-null value, even "", means the component will show in an error state.